src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -298,5 +300,39 @@ public override string ToString()
             return InternalFamilyInstance.Name;
         }
 
+       #region Public Methods
+
+        /// <summary>
+        /// Sets the Euler angle of the family instance around its local Z-axis.
+        /// </summary>        
+        /// <param name="degree">The Euler angle around Z-axis.</param>
+        /// <returns>The result family instance.</returns>
+        public FamilyInstance SetRotation(double degree)
+        {
+           if (this == null)
+              throw new ArgumentNullException("familyInstance");
+
+           TransactionManager.Instance.EnsureInTransaction(Document);
+
+           // Rotate the element.
+           var oldTransform = InternalFamilyInstance.GetTransform();
+           double[] oldRotationAngles;
+           TransformUtils.ExtractEularAnglesFromTransform(oldTransform, out oldRotationAngles);
+
+           Double newRotationAngle = degree * Math.PI / 180;
+
+           if (!oldRotationAngles[0].AlmostEquals(newRotationAngle, 1.0e-6))
+           {
+              double rotateAngle = newRotationAngle - oldRotationAngles[0];
+              var axis = Line.CreateUnbound(oldTransform.Origin, oldTransform.BasisZ);
+              ElementTransformUtils.RotateElement(Document, new ElementId(Id), axis, -rotateAngle);
```

Comment:
We define the rotation is always along its local Z-axis, this is why we removed the argument rotation_axis. Does it make sense?